### ISSUE TITLE
appveyor failing after merging #130

### ISSUE THREAD

**Issue by jgabry** [stan-dev/cmdstanr#132]
@rok-cesnovar Any idea why the latest appveyor build is failing? The branch for #130 passes (https://ci.appveyor.com/project/jgabry/cmdstanr/builds/31128819) but the build after I merged the PR to master fails (https://ci.appveyor.com/project/jgabry/cmdstanr/builds/31185234).

**Comment by jgabry** [comment]
weird, now it seems to be passing again after i made some doc commits that shouldn't have changed anything

**Comment by rok-cesnovar** [comment]
Yeah, cmdstan develop was failing yesterday morning and afternoon CET time. 
@serban-nicusor-toptal notified me about this in the afternoon and we got to the bottom of it and Nic has fixed it in the evening (afternoon ET). That is why the latest test then passed.

The reason was that there was no nightly build of stanc3 on which develop cmdstan relies on. The nightly build failed due to some one-off CI Jenkins network issues (see https://jenkins.mc-stan.org/blue/organizations/jenkins/stanc3/detail/master/604/pipeline/). Nic sorted all this out and everything is back to normal.

It slipped my mind that cmdstan develop failing also causes issues for cmdstanr tests, otherwise I would have notified you.

Closing.
